main/res/values/strings.xml

```diff
@@ -727,7 +727,7 @@
     <string name="downloader_cancelled_download">Cancelled download for "%s"</string>
     <string name="downloader_pending_info">There is at least one pausing or failed download. Do you want to open the list of pending downloads?</string>
     <string name="downloader_retry_download">Retry download</string>
-    <string name="downloader_retry_download_details">Do you want to retry the download of</string>
+    <string name="downloader_retry_download_details">Please select the files you want to download again:</string>
```

Comment:
There will always be only a single file, as this dialog pops up as soon as you hit the "retry" button in the view.

(This differs from other usages of the same dialog, eg. after performing an update check or routing data coverage check, and offering one or multiple files to download. In that case, a different info text is displayed, though.)

Reply:
OK. Then probably "Do you want to retry the download of the following file?"